src/portfolio/APIClient.ts

```diff
@@ -96,6 +97,30 @@ export type PositionQty = {
   qty: number;
 };
 
+export type Backtesting = {
+  MDDMaxToMin: number;
+  MDDInitialToMin: number;
+  dateMax: string;
+  dateMin: string;
```

Comment:
Everywhere else we have UNIX timestamps in API responses and `Date` objects in the frontend application.

It's fine if we get the dates as strings here, but we should clarify and document the date format used in the strings and convert it to a `Date` object so we can display local date strings (as for next dividend date).

Reply:
So should I rename this type to BacktestingResponse and create another one for the rest of the app called Backtesting as well as a converter function ?

Reply:
That's how I did it for the other objects where a conversion is necessary